aws/resource-count-aws.sh

```diff
@@ -68,6 +68,39 @@ else
    WITH_DATA="false"
 fi
 
+
+##########################################################################################
+## Optionally pass an AWS profile to use. Profiles are available at ~/.aws/config (Linux & Mac) or %USERPROFILE%\.aws\config (Windows)
+##########################################################################################
+
+echo ">>> Profiles are available at ~/.aws/config (Linux & Mac) or %USERPROFILE%\.aws\config (Windows)"
```

Comment:
Documenting the implementation of AWS Profiles is outside the scope of this script.